### PR TITLE
adding nodes

### PR DESCRIPTION
**Pull request by joel-hamill**

### CODE REVIEW

**Comment by surdy** [comment]
Looks good 👍

File: 1.8/administration/installing/custom/add-a-node.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    $ sudo bash /opt/dcos_install_tmp/dcos_install.sh slave
+    ```
+    
+    Public agent nodes:
+    
+    ```bash
+    $ dcos node --json | jq --raw-output '.[] | select(.reserved_resources.slave_public != null) | .id' | wc -l
+    ```
+    
+    **Tip:** You can determine the node type by running this command from the DC/OS CLI.
```
Comment (emanic) [inline 1.8/administration/installing/custom/add-a-node.md]: Not sure why you would want to determine the node type at this juncture. Perhaps this is to verify that your installation succeeded?

File: 1.8/administration/installing/custom/add-a-node.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+You can add agent nodes to an existing DC/OS cluster. 
+
+Agent nodes are designated as [public](/docs/1.8/overview/concepts/#public) or [private](/docs/1.8/overview/concepts/#private) during installation. By default, they are designated as private during [GUI][1] or [CLI][2] installation.
+
+### Prerequisites:
+
+*   DC/OS is installed using the [custom](/docs/1.8/administration/installing/custom/) installation method.
+*   The archived DC/OS installer file (`dcos-install.tar`) from your [installation](/docs/1.8/administration/installing/custom/gui/#backup).
+*   Available agent nodes that satisfy the [system requirements](/docs/1.8/administration/installing/custom/system-requirements/).
+*   The CLI JSON processor [jq](https://github.com/stedolan/jq/wiki/Installation).
```
Comment (emanic) [inline 1.8/administration/installing/custom/add-a-node.md]: Perhaps another prerequisite is to be set up for SSH access?

File: 1.8/administration/installing/custom/add-a-node.md (hunk 1 above)
Comment (joel-hamill) [inline 1.8/administration/installing/custom/add-a-node.md]: @emanic yeah, basically to verify it worked. i'll add some more explanation.

Comment (joel-hamill) [inline 1.8/administration/installing/custom/add-a-node.md]: changed to `You can verify the node type by running this command from the DC/OS CLI`

**Pull request merged**